examples/java/pom.xml

```diff
@@ -432,6 +424,11 @@
 
     <dependency>
       <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-direct-java</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.beam</groupId>
       <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
```

Comment:
I've noticed that, in fact, the `include-runners` profile is active by default. So all the runners are there unless other profile shenanigans disable this one.

We should still split the profile into many because runners may have diamond dependency conflicts that shouldn't require solving in order to run these examples.

I've added sub-profiles so when integration tests are updated we can delete the bulk profile.